sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -168,6 +168,105 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
       Row(1, 1) :: Row(1, 2) :: Row(2, 1) :: Row(2, 2) :: Row(3, 1) :: Row(3, 2) :: Nil)
   }
 
+  test("support star except") {
```

Comment:
these end-to-end tests should all go to the golden files.

Reply:
I have similar stuff in golden files. Are you suggesting that I revert these ?

Reply:
Yes